src/PageContainer.js

```diff
@@ -19,6 +20,13 @@ class PageContainer extends Component {
 
         this.state = {
             content: props.content,
+            showPopUp: false,
+            hover: false,
```

Comment:
nit-pick: rename `hover` to something like `notificationClicked`? since it's really on click not hover